manifests/install.pp

```diff
@@ -4,7 +4,7 @@
 class candlepin::install {
   assert_private()
 
-  package { ['candlepin']:
+  package { ['candlepin', 'candlepin-selinux']:
```

Comment:
Do the exact version numbers match? Also note there's a puppet bug which means Pupppet doesn't reload any selinux types it installed during the run so some file types might not be set right.

Reply:
They appear to on 3.8:

```sh
$ rpm -q candlepin-selinux candlepin
candlepin-selinux-2.4.6-1.el7.noarch
candlepin-2.4.6-1.el7.noarch
```

Reply:
Agree they match. candlepin-selinux is a subpackage with same version: http://koji.katello.org/koji/buildinfo?buildID=37128